modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
gui/vizkit is the particular example triggering that addition -- here rock_ruby_rice_extension macro is being used, but that will require adapting the RUBY_EXTENSION_INSTALL_DIR or RUBY_EXTENSION_LIBRARY_DIR directly to achieve the same result